frontend/poi.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     require('poi-preset-babel-minify')(),
     require('poi-preset-webpackmonitor')()
```

Comment:
btw they are poi 9 presets, for poi 10 it should be `@poi/plugin-babel-minify`

https://poi.js.org/migrate-from-9-to-10.html